README.md

```diff
@@ -92,6 +92,10 @@ make test
 
 ## CLI
 
+> [!WARNING]
```

Comment:
I didn't experience something not working with the CLI itself

Reply:
See my other comment about `cast` limiting argument size. This may actually be a limitation on the OS, which limits argument sizes by default? This is non-critical, so @claudioantonio and I agreed to have this broken for the time being, and fix it on a later PR. I think the way is to mimic how EggRoll was creating and signing transactions using `go-ethereum`.

Reply:
Yes. It's a system issue. Try increase stack space limit using `ulimit -s` as mentioned [here](https://github.com/crypto-bug-hunters/bugless/issues/76#issuecomment-2060509813).